docsource/modules170-180.rst

```diff
@@ -988,7 +988,7 @@ Module coverage 17.0 -> 18.0
 +---------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| spreadsheet_dashboard_purchase              |                      |                                                 |
 +---------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| spreadsheet_dashboard_purchase_stock        |                      |                                                 |
+| |del| spreadsheet_dashboard_purchase_stock        |                      |Moved to Enterprise                              |
```

Comment:
Does they really move them to enterprise?

Reply:
Ouch... I hope they don't block that we rescue it from previous versions in OCA.

Reply:
Not possible this time. It's not an error in a commit in an unreleased version.